files/main.sh

```diff
@@ -28,11 +28,11 @@ setup_server () {
 
 setup_pkcs11 () {
 	[ "${CRYPTOSERVER:-}" != "" ] || die 2 "Missing CRYPTOSERVER env variable."
-	sed -i'' -e "s|CRYPTOSERVER|${CRYPTOSERVER}|" /opt/utimaco/p11/libcs_pkcs11_R2.cfg
```

Comment:
Again, this should be able to handle both PKCS11 engines at the same time.. not replacing one by the other.